src/components/organism_word_card_chunk/index.tsx

```diff
@@ -22,6 +22,7 @@ const WordCardsChunk: FC = () => {
   return (
     <StyledSuspense>
       <WordCardChunkSearchFound />
+      {wordIds.length} word cards
```

Comment:
```suggestion
      <Typography variant="caption">
        {wordIds.length} word cards
      </Typography>
```

Use typography to make the font style consistent.
https://mui.com/material-ui/react-typography/#usage

Typography supports many font variants
@akanex115  TODOs
- [ ] Test different variants in your own machine
![image](https://github.com/ajktown/wordnote/assets/53258958/8efea5ce-5418-4968-925e-573913a713ad)